### ISSUE TITLE
I cant Export project

### ISSUE THREAD

**Issue by digitonAV** [google/marzipano#83]
I was doing a few 360 Views and now I cant seem to make anymore. Any help?

**Comment by manuelcabral** [comment]
Can you provide some more details? Is there an error?

**Comment by trcorp1** [comment]
when loading the marzipano tool it generates these errors:

GET http://www.marzipano.net/tool/template/webserver-windows.exe?45c6dea 403 (Forbidden)
app.js?45c6dea:14 Uncaught Error: Error loading template/webserver-windows.exe?45c6dea via XHR
    at XMLHttpRequest.i.onload (app.js?45c6dea:14)

When choosing export you get this error: 

app.js?45c6dea:10 Uncaught Error: Template files not loaded
    at r (app.js?45c6dea:10)
    at n (app.js?45c6dea:10)
    at Object.x (app.js?45c6dea:10)
    at HTMLAnchorElement.<anonymous> (app.js?45c6dea:7)

**Comment by trcorp1** [comment]
looks like this may be an issue with an aggressive content filter. 
![capture](https://user-images.githubusercontent.com/29378296/27040726-831f3e8a-4f57-11e7-8874-266cf50d54d4.PNG)
perhaps doesn't like the .exe

**Comment by digitonAV** [comment]
whats the best way to solve it?

**Comment by trcorp1** [comment]
On the office I haven't been able to get around it. Submitted request to have it exempted but haven't heard back yet. As a quick fix I took the laptop over to Starbucks. Wasn't blocked on their network so it worked just fine.

**Comment by tjgq** [comment]
This is a duplicate of #93.
